src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -120,22 +119,20 @@ class PaymentsPage extends React.Component {
                         onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
                         onCloseButtonPress={() => Navigation.dismissModal(true)}
                     />
-                    <ScrollView style={styles.flex1}>
-                        {
-                            Permissions.canUseWallet(this.props.betas) && <CurrentWalletBalance />
-                        }
-                        <Text
-                            style={[styles.ph5, styles.formLabel]}
-                        >
-                            {this.props.translate('paymentsPage.paymentMethodsTitle')}
-                        </Text>
-                        <PaymentMethodList
-                            onPress={this.paymentMethodPressed}
-                            style={[styles.flex4]}
-                            isLoadingPayments={this.props.isLoadingPaymentMethods}
-                            isAddPaymentMenuActive={this.state.shouldShowAddPaymentMenu}
-                        />
-                    </ScrollView>
+                    <PaymentMethodList
+                        ListHeaderComponent={() => (
```

Comment:
Just to confirm, this header is still sticky right? As in if we have a ton of accounts and the user scrolls it's not going to scroll too?

Reply:
Oh is the header (Account balance + Payments method text) supposed to be sticky? I thought it was supposed to scroll since it scrolls in the current implementation.

Reply:
+1 

Could we also just put the header outside of the `FlatList` if we are worried about this?

Reply:
Current behavior:
https://user-images.githubusercontent.com/22219519/148615425-d1e165b6-7e60-4841-a75f-e4254423bf1b.mov

Reply:
>Could we also just put the header outside of the FlatList if we are worried about this?
Yes, the solution is simpler if the header is supposed to be sticky. I just thought it had to scroll since that's how ti behaves now.

Reply:
Awesome! Thanks